packages/selleo-design-core/src/LoginView.tsx

```diff
@@ -0,0 +1,156 @@
+import { h } from 'preact';
+
+import { Button } from './Button';
+import { Card } from './Card';
+import {
+  FacebookIcon,
+  GithubIcon,
+  GoogleIcon,
+  SelleoLogoIcon,
+  TwitterIcon,
+} from './icons';
+import { Input } from './Input';
+
+export function LoginWithPasswordView() {
+  return (
+    <Card>
+      <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
```

Comment:
Can that be merged, so Card Just takes those styles as classaName?

Reply:
Yes, I'll merge it, I've just added possibility to add additional styles to the Card component and it takes className as a prop to add own styles :D